python-threatexchange/threatexchange/cli/label.py

```diff
@@ -22,11 +22,8 @@ class LabelCommand(command_base.Command):
     matches by default in the match command.
 
     Examples:
-      # Label text
-      $ threatexchange -c te.cfg label violating_label_from_config,other_label text "this is an example bad text"
-
       # Label descriptor
-      $ threatexchange -c te.cfg label false_positive descriptor 12345
+      $> threatexchange -c te.cfg label,other_label false_positive descriptor 12345
```

Comment:
```suggestion
      $> threatexchange -c te.cfg label false_positive,other_label descriptor 12345
```

I believe you meant this change, correct?